### PR TITLE
Use vendor dir when building

### PR DESCRIPTION
**Pull request by oskarth**
See https://github.com/status-im/status-console-client/issues/22

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,7 +3,7 @@ SHELL := /bin/bash
 GO111MODULE = on
 
 build:
-	go build -o ./bin/status-term-client .
+	GOFLAGS=-mod=vendor go build -o ./bin/status-term-client .
```
Comment (adambabik) [inline Makefile]: It won't be possible to replace it if wrote in this way. This should work the same but one can override it like this: `GOFLAGS=-some-other make build`.

```
build: GOFLAGS=-mod=vendor
build:
	go build -o ./bin/status-term-client .
.PHONY: build
```

Comment (oskarth) [reply Makefile]: Oops you are right, I just did a `make build` and didn't think further. Your snippet didn't work but this seems to. Pushing fix.

```
[oskarth@localhost status-console-client]$ GOFLAGS=-some-other make build
GOFLAGS=-some-other go build -o ./bin/status-term-client .
go: parsing $GOFLAGS: unknown flag -some-other
make: *** [Makefile:7: build] Error 1
[oskarth@localhost status-console-client]$ make build
GOFLAGS="-mod=vendor" go build -o ./bin/status-term-client .
```

Comment (oskarth) [reply Makefile]: N.B. My Makefile-fu isn't great, so YMMV with C/GNUMake etc

**Comment by oskarth** [comment]
Didn't sign first time around, squashed and signed

**Pull request merged**